ts/screens/wallet/payment/TransactionSummaryScreen.tsx

```diff
@@ -93,8 +92,6 @@ const NOTICE_ICON_SIZE = 24;
  */
 class TransactionSummaryScreen extends React.Component<Props> {
   public componentDidMount() {
-    // refresh wallet list (to avoid cache de-sync)
-    this.props.dispatchWalletsLoad();
```

Comment:
Why this operation is no more needed?

Reply:
I added this refresh in the previous PR https://github.com/pagopa/io-app/pull/2837/files
but I found some strange behaviors when the wallet is empty. So I restore it as it was